src/components/ReposDetails/ReposDetails.tsx

```diff
@@ -0,0 +1,20 @@
+interface RepoDetailsProps {
+    name: string;
+    visibility: string;
+    description: string;
+}
+
+function ReposDetails({ name, description, visibility }: RepoDetailsProps) {
+    return (
+        <>
+            <h1 className="red">Repo: {name} </h1>
+            <h2 className='red'>
+                {visibility}
+                {description}
+            </h2>
+            <br />
```

Comment:
Avoid using `<br/>` tags, it inserts too much unexpected behavior style-wise. If you need space in the bottom create a new class to do it.